contracts/MoMABurn.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+import "./FeralfileArtworkV3_3.sol";
+
+contract MoMABurn is Ownable {
+    using SafeMath for uint256;
+    address constant DEAD_ADDRESS = 0x000000000000000000000000000000000000dEaD;
+
+    ERC721 public burnContract; // Address of token need to burn
+    FeralfileExhibitionV3_3 public tier1Contract; // Address of new token for Tier 1
+    FeralfileExhibitionV3_3 public tier2Contract; // Address of new token for Tier 2
+    FeralfileExhibitionV3_3 public tier3Contract; // Address of new token for Tier 3
+
+    uint256 private tier1ArtworkID; // Artwork ID of Tier 1
+    uint256 private tier2ArtworkID; // Artwork ID of Tier 2
+    uint256 private tier3ArtworkID; // Artwork ID of Tier 3
+
+    bool public isBurnEnabled;
+
+    constructor() {
+        isBurnEnabled = false;
```

Comment:
what about set the 
- burnContractAddress
- tier1Address
- tier2Address
- tier3Address
- tier1ArtworkID
- tier2ArtworkID
- tier3ArtworkID
on deployment?

Reply:
Added those params to constructor